docs/hello_nextflow/01_hello_world.md

```diff
@@ -320,25 +320,25 @@ It allows us to retrieve the desired output file without having to dig down into
 
 In the workflow script file `hello-world.nf`, make the following code modification:
 
-_Before:_
+=== "After"
 
-```groovy title="hello-world.nf" linenums="6"
-process sayHello {
+    ```groovy title="hello-world.nf" linenums="6" hl_lines="2 3 4"
```

Comment:
I don't think that we should highlight surrounding lines, only the specific lines that are relevant.

Reply:
like a diff you mean?

Reply:
Makes sense! I will fix that.